content/english/secret-messages/activity-4.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Activity 4 - Conditions"
+date: 2022-2-23
+weight: 4
+draft: true
+---
+
+Very well!
+
+We are getting closer and closer to making our secret messages. But also, let's not forget that we have to be able to decipher those messages!
+
+For that, we are going to enter in the console a 1, if we want to encrypt a message, or the number 2, if we want to decrypt it.
+
+For this we will need the CONDITIONS.
+
+{{% notice tip %}}
+
+### Conditions if / else
+A condition in programming is an instruction or a group of instructions, which can be executed or not, based on whether a condition is true. It would look something like this: **IF**(if) this happens, execute these instructions, **IF NOT** (else), execute these other instructions.
+
+Example
+...
+if(3 > 1){
+  cout << "3 es mayor que 1" << endl;
+}else{
+  cout << "3 no es mayor que 1" << endl;
+}
+...
+The program will print:
+...
+3 es mayor que 1
+...
+
+
+There we are indicating that if the number 3 is greater than 1 '(3>1)', let what is between its keys be executed '{}'. And otherwise, let what is in the block be executed as 'else'. 
```

Comment:
```suggestion
Here we are indicating that if the number 3 is greater than 1 '(3>1)', execute the code between the first set of '{}'. Otherwise, execute the code that appears between the set of '{}' after the word 'else'. 
```